cypress/support/commands.js

```diff
@@ -23,3 +23,10 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('testSetup', () => {
```

Comment:
A custom Cypress command that contains all the test setup we need to run in a `beforeEach`.